components/Icon.js

```diff
@@ -114,13 +113,13 @@ const Icon = ({
     style,
     viewBox: icon.viewBox,
     width: width || icon.width,
-    ...other,
+    ...other
   };
 
   const svgContent = icon ? svgShapes(icon.svgData) : '';
 
   return (
-    <svg {...props}>
+    <svg {...props} aria-label={description}>
```

Comment:
Including `aria-label` with `description` prop to satisfy a11y issue raised by aXe.
According to [StackOverflow](https://stackoverflow.com/a/4756461/2058360):

![image](https://user-images.githubusercontent.com/4185382/28580322-c0c15022-7124-11e7-8bc3-b0b07c48ecf3.png)